run.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+# Declare default type
+type="gcp-pubsub"
+
+
+# Prints usage
+# Output:
+#   Help usage
+function show_help () {
+    echo -e "Usage: ./run.sh --listener_url=<listener_url> --token=<token> --region=<region> --type=<type>"
+    echo -e " --listener_url=<listener_url>           Logz.io Listener URL (You can check it here https://docs.logz.io/user-guide/accounts/account-region.html)"
+    echo -e " --token=<token>                     Logz.io token of the account you want to ship to."
+    echo -e " --region=<region>                   Region where you want to upload Cloud Funtion."
+    echo -e " --type=<type>                       Log type. Help classify logs into different classifications"
+    echo -e " --help                              Show usage"
+}
+
+# Gets arguments
+# Input:
+#   Client's arguments ($@)
+# Output:
+#   listener_url - Logz.io Listener URL
+#   token - Logz.io token of the account user want to ship to.
+#   region - Region where user want to upload Cloud Funtion.
+#   type -  Log type. Help classify logs into different classifications
+# Error:
+#   Exit Code 1
+
+function get_arguments () {
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Getting arguments ..."
+    while true; do
+        case "$1" in
+            --help)
+                show_help
+                exit
+                ;;
+            --listener_url=*)
+                listener_url=$(echo "$1" | cut -d "=" -f2)
+                if [[ "$listener_url" = "" ]]; then
+                    echo -e "\033[0;31mrun.sh (1): no Logz.io Listener URL specified!\033[0;37m"
```

Comment:
Please change the message to a more informative one. Something like:
Logz.io Listener URL is missing please rerun the script with the relevant parameters.
Also please change for the other error messages